### PR TITLE
Added a basic creature class

### PR DESCRIPTION
**Pull request by x183**

### CODE REVIEW

File: core/src/com/g16/feyrune/creature/BaseCreature.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.g16.feyrune.creature;
+
+import com.badlogic.gdx.math.Vector2;
+
+public class BaseCreature {
+    //Stats variables
+    private double health;
+    private double damage;
+    private double movementSpeed;
+    private boolean aggressive;
```
Comment (Supergamer1337) [inline core/src/com/g16/feyrune/creature/BaseCreature.java:10]: Why do this thing here exist? When are we supposed to use this?

Comment (Supergamer1337) [reply core/src/com/g16/feyrune/creature/BaseCreature.java:10]: Specifically `boolean aggressive`

**Review by x183 — COMMENTED** [review]
removed agressive

**Review by Krysset — APPROVED** [review]
Very good 👍🏻

**Pull request merged**